domain/src/main/java/com/camp/nextstep/edu/domain/Operator.kt

```diff
@@ -0,0 +1,36 @@
+package com.camp.nextstep.edu.domain
+
+import com.camp.nextstep.edu.domain.Operator.TokenType.*
+
+class Operator {
+
+    sealed class TokenType(val operator: String)  {
```

Comment:
Operator 내부에 TokenType 클래스를 따로 만든 이유가 있을까요? 
Operator로 감싸지 않아도 될거 같아요!
```suggestion
sealed class Operator(val operator: String) {
```

Reply:
단독 클래스로 구분하였습니다.